mule-extensions-api/src/main/java/org/mule/runtime/extension/api/stereotype/ConfigurationStereotype.java

```diff
@@ -17,7 +17,7 @@
 
   @Override
   public String getName() {
-    return "CONFIGURATION";
+    return "APP_CONFIGURATION";
```

Comment:
this one is misleading. Does it refer to the app config as a whole?, does it refer to a module config inside an app? does it refer to `<mule:configuration>`?

Reply:
`<mule:configuration>`